default/python/AI/InvasionAI.py

```diff
@@ -284,6 +286,12 @@ def evaluate_invasion_planet(planet_id, empire, secure_fleet_missions, verbose=T
         bld_tally += bval
         detail.append("%s: %d" % (bldType, bval))
 
+    tech_tally = 0
+    for unlocked_tech in AIDependencies.SPECIES_TECH_UNLOCKS.get(species_name, []):
+        if not tech_is_complete(unlocked_tech):
+            rp_cost = fo.getTech(unlocked_tech).researchCost(empire_id)
+            tech_tally += rp_cost * 10
```

Comment:
Valuing the tech at 10 times research cost seems a bit high to me.  Generally I was valuing research points at 2-3 PP each, and so it seems to me that a similar multiplier should be used here.